app/models/scc_account.rb

```diff
@@ -230,4 +254,32 @@ def update_scc_products(upstream_products)
       end
     end
   end
+
+  # validate the subscription status of a product/repo
+  # no saving to database
+  # params: elements: scc repos or products, Array or ActiveRecord_(*)
+  #         scc_id: scc_id of the element that should be revalidated
+  # return: elements where for the element with scc_id subscription_valid is true
+  def revalidate_subscription_status(elements, scc_id)
+    return if elements.nil?
```

Comment:
This will probably return `nil` implicitly.
I would prefer to explicitly `return nil`, because it makes it more readable.

Reply:
Fixed.